### DISCUSSION TITLE
Low light image quality

### DISCUSSION THREAD

**Discussion by Orsbo-byte** [gopro/labs#232] [General]
Hi!

I'm doing an indoor timelapse sequence at a construction site. The lighting is somewhat poor, but I thought it would be ok. As I downloaded the first footage the image quality was rather bad, much worse than I anticipated. 

My theory is that this could be enhanced by using longer exposure times and lower ISO settings, is this correct?  As the camera is already in place, I can't experiment myself (I'll have to fly a QR up to the camera with a drone to reprogram it, but that's another story..)

Which tweaks and settings should I use in your opinion? Will I be able to change the shutter time in timelapse-mode or do I need to switch to nightlapse?

Super thankful for any help! 

Best regards

Martin

**Comment by dnewman-gpsw** [comment]
Always use night lapse, even for day time projects, if the camera is stationary.  Night lapse auto will bias to long shutter, not higher gain, but will not overexpose in daytime conditions.
